pkg/kubelet/kubelet.go

```diff
@@ -2207,8 +2214,8 @@ func (kl *Kubelet) updateRuntimeUp() {
 	runtimeReady := s.GetRuntimeCondition(kubecontainer.RuntimeReady)
 	// If RuntimeReady is not set or is false, report an error.
 	if runtimeReady == nil || !runtimeReady.Status {
-		err := fmt.Errorf("Container runtime not ready: %v", runtimeReady)
-		klog.Error(err)
+		err := fmt.Errorf(runtimeReady.String())
+		klog.ErrorS(err, "Container runtime not ready")
```

Comment:
is there a recommended way to transform a non-ready condition into the error?

Reply:
Same as above